data_model/src/pagination.rs

```diff
@@ -70,7 +72,7 @@ impl<I: Iterator> Iterator for Paginated<I> {
 }
 
 /// Structure for pagination requests
-#[derive(Clone, Eq, PartialEq, Debug, Default, Copy, Deserialize, Serialize)]
+#[derive(Clone, Eq, PartialEq, Debug, Default, Copy, Deserialize, Serialize, IntoSchema)]
```

Comment:
if other structures are referencing this how come `no_missing_schemas` test didn't catch this? if not then you should add `Pagination` to `build_schemas()`. Also, if it's not referenced through other structures why is it's schema necessary?

Reply:
It's necessary because SDKs we're not able to use this structure for submitting paginated queries.

Reply:
A different question is why, it wasn't included in the missing schemas test,

Reply:
@mversic `Pagination` is one of top-level types that HTTP requests can contain as a query param  
@appetrosyan `no_missing_schemas` just covers types under the top-level types listed in `build_schemas()`

Reply:
It should then be listed in `build_schemas`

Reply:
all looks good, you just need to replace `VersionedQueryResult` in `build_schemas` with `VersionedPaginatedQueryResult`

Reply:
The same from me. Also please modify `api_spec.md`